src/features/web-compat.js

```diff
@@ -512,15 +512,42 @@ export default class WebCompat extends ContentFeature {
     }
 
     viewportWidthFix () {
-        const viewportTag = document.querySelector('meta[name=viewport]')
-        if (!viewportTag) return
-        const viewportContent = viewportTag.getAttribute('content')
-        if (!viewportContent) return
-        const viewportContentParts = viewportContent.split(',')
-        const widthPart = viewportContentParts.find((part) => part.includes('width'))
-        // If we already have a width, don't add one
-        if (widthPart) return
-        viewportTag.setAttribute('content', `${viewportContent},width=device-width`)
+        const viewportTags = document.querySelectorAll('meta[name=viewport]')
+        // Chrome respects only the last viewport tag
+        let viewportTag = viewportTags.length === 0 ? null : viewportTags[viewportTags.length - 1]
+        const viewportContent = viewportTag?.getAttribute('content')
+        const viewportContentParts = viewportContent?.split(/,|;/)
+        if (!viewportTag || this.desktopModeEnabled) {
+            // force wide viewport width
+            if (!viewportTag) {
+                viewportTag = document.createElement('meta')
+                viewportTag.setAttribute('name', 'viewport')
+            }
+            const forcedWidth = screen.width >= 1280 ? 1280 : 980
+            // Race condition: depending on the loading state of the page, initial scale may or may not be respected, so the page may look zoomed-in after applying this hack.
+            // Usually this is just an annoyance, but it may be a bigger issue if user-scalable=no is set, so we remove it too.
+            const forcedInitialScale = (screen.width / forcedWidth).toFixed(3)
+            const newContentParts = [`width=${forcedWidth},initial-scale=${forcedInitialScale}`]
+            viewportContentParts?.forEach((part) => {
+                if (!part.includes('width') && !part.includes('initial-scale') && !part.includes('user-scalable')) {
```

Comment:
Should we care about the value here do you think?
What about preserving ordering of the width, initial scale too?

```suggestion
                const [key, value] = part.split('=').map(a => a.trim())
                if (!['width', 'initial-scale', 'user-scalable'].includes(key)) {
```

Reply:
The order should not matter since we remove all original occurrences, so it's guaranteed that there will only be one occurrence in the final value.
Searching within the key only makes sense. And it should be case-insensitive btw, I'll make the change